software/spmd/bp_manycore_test/bp_main.c

```diff
@@ -0,0 +1,164 @@
+#include <stdint.h>
```

Comment:
Add bp_manycore_test to NO-RECURSE. https://github.com/bespoke-silicon-group/bsg_manycore/blob/eb132d1915350ac79df451b1a7bd37caf5874ce3/software/spmd/Makefile#L9